forge-game/src/main/java/forge/game/ability/effects/DamageEachEffect.java

```diff
@@ -39,12 +41,7 @@ protected String getStackDescription(SpellAbility sa) {
             sb.append(sa.getParam("StackDescription"));
```

Comment:
This first if shouldn't be needed...
StackDescription would always override anyway?